aws_s3_rules/aws_s3_unknown_requester_get_object.yml

```diff
@@ -3,7 +3,7 @@ Filename: aws_s3_unknown_requester_get_object.py
 RuleID: AWS.S3.ServerAccess.UnknownRequester
 DisplayName: AWS S3 Unknown Requester
 DedupPeriodMinutes: 60 # 1 hour
-Enabled: true
+Enabled: false
```

Comment:
Any time we ship with a rule turned off by default, it should have the 'Configuration Required' tag

Reply:
Are we disabling this because it's too noisy, requires configuration, or has been deprecated? If it's because it requires configuration then definitely let's add the tag, if its noisy we should tune it, if its deprecated we should indicate that.

Reply:
I was disabling it because of the existing "configuration required" tag.   Technically it works as-is, but does require configuration for someone that wants to monitor s3 buckets other than `panther-bootstrap-processeddata-*` I can re-evaluate / re-enable.

 More details about the noisy detections are in the task comments in asana